src/app/common/resolvers/active-tab.resolver.ts

```diff
@@ -6,9 +6,24 @@
  */
 
 import { ActivatedRouteSnapshot, ResolveFn, RouterStateSnapshot } from "@angular/router";
+import { DatasetViewTypeEnum } from "src/app/dataset-view/dataset-view.interface";
+import ProjectLinks from "src/app/project-links";
 
-export const activeTabResolver: ResolveFn<string> = (_: ActivatedRouteSnapshot, state: RouterStateSnapshot) => {
+export const activeTabResolver: ResolveFn<string> = (route: ActivatedRouteSnapshot, state: RouterStateSnapshot) => {
```

Comment:
The name is confusing. It feels like it can be applied to any set of tabs, but it's very specified to ``/<account>/<dataset>/...`